events/models.py

```diff
@@ -27,6 +33,7 @@ class Event(models.Model):
     name = models.CharField(max_length=50, help_text=_("Event Name"))
     venue = models.ForeignKey(Address, on_delete=models.CASCADE)
     center = models.ForeignKey(Center, on_delete=models.CASCADE, help_text=_("Center"))
+    category = models.ForeignKey(EventCategory, on_delete=models.CASCADE, help_text=_("Event Category"), default="")
```

Comment:
should the foreign key have a default of empty string ?

Reply:
Shouldn't have, but this is creating issues for existing records...so added this...do you know of a better way to handle this ?

Reply:
A better way would be the set the default to null for existing entries ? Anyways, it's fine for now :)